ting_search.module

```diff
@@ -973,5 +973,13 @@ function ting_search_submit($form, &$form_state) {
  */
 function _ting_search_quote($string) {
   $cqlDoc = new TingSearchCqlDoctor($string);
-  return $cqlDoc->string_to_cql($string);
+  $return = $cqlDoc->string_to_cql();
+
+  // Issue 1392 filter on agency if required
```

Comment:
Please write short description, so developers don't have to look up why this code exists.